main.js

```diff
@@ -0,0 +1,123 @@
+"use script";
+var counters = {};
+var arr = [];
+var btnSort = document.getElementById('sort'); 
+var startContainer = document.getElementById('start-container');
+var endContainer = document.getElementById('end-container');
+var blockRes = document.getElementById('res');
+var ul = document.createElement('ul');
+var btn = document.getElementById('btn');
+
+//Функция для формирования массива с фильтрованными данными. 
+function getArr(){
+    var str = btnSort.value;
+    if(!str){alert('Заполните поле!'); return;}
+    str = str.replace(/ /g, "")
+    //Убираем с конца запятую если есть.
+    if(str.slice(-1) == ','){
+        str = str.slice(0, -1);
+    }
+    var arr1 = str.split(','); 
+     for(var i = 0; i < arr1.length; i++){
+        if(!isNaN(parseFloat(arr1[i])) && isFinite(arr1[i])){
+             arr.push(+arr1[i]);
+             continue;
+        }
+             alert("Упс! <" + arr1[i] + ">  Не число! Или вы ввели лишние знаки!");
+             return false;
+    }
+    loadRes();
+}
+
+//Функция формирования страницы для введенных чисел и их сортировки.
+function loadRes(){
+      startContainer.className = "hide";
+      endContainer.className = "";
```

Comment:
And what if these containers had more than one style class? You.d better use classList.add and classList.remove